financial-services/1.10/modules/ROOT/pages/rpa-setup-guide.adoc

```diff
@@ -0,0 +1,183 @@
+
+xref:rpa-manager::processautomation-deploy.adoc#deploy-a-process-automation[Deploy the project] and schedule it as needed.
+
+
+[[example-of-adapting-the-crpa-file]]
+== Example of Adapting the CRPA File
+
+[NOTE]
+The example in this guide uses real world data to show you how to adapt a CRPA file to a specific AMS.
+
+All values for the Activity Parameters used in RPA Builder are test values that you must overwrite with your own values when creating the run configuration for the process in RPA Manager.
+
+=== Process Model
+
+The process model contains the following three activities:
+
+. Log in to the AMS using credentials and a database name.
+. Open a record in the AMS database using a lookup code.
+. Copy data from various fields on the record pages to a CSV file.
+//Step 3 above should be a CSV file but is a POST REST call in JSON format to http://mainstreet-demo.us-e2.cloudhub.io/epicpolicy. I (Tasha) wrote the instructions as a CSV. What should the text actually state?
+
+image::rpa-setup-process-model.png[RPA Builder process model]
+
+You can add activities by clicking on an element on the model panel and choosing an element from the context menu. For example, the following image shows how to branch off to a second AMS if you are using more than one and not finding the data you need in the first AMS:
+
+image::rpa-setup-add-activity.png[Add activity to RPA Builder process model]
+
+Click on an activity to open the editing workflow.
+
+=== Activity 1: Log in to the AMS
+
+==== Overview of the Activity Workflow
+
+. Decrypt the AMS login credentials that go in through an Activity Parameter.
+. Run the AMS.
+. Log in to the AMS using the decrypted credentials.
+. Select the database using another in-going Activity Parameter.
++
+image::rpa-setup-activity-parameters-wizard.png[RPA Builder Activity Parameter Wizard]
+
+==== Minimum Steps to Adapt in This Workflow
+
+. Change the following in the xref:rpa-builder::toolbox-control-run-program.adoc[Run Program Action Step]:
+.. Replace the values in the *File name*, *Parameters*, and *Window Title* fields with the values from your AMS.
+.. Click the *Test* button to run the application so that you can use it to adapt the following Action Steps:
++
+image::rpa-setup-run-program-wizard.png[RPA Builder Run Program Wizard]
+
+. Use the *Start from scratch* option to capture and test a new pattern that indicates the login page of the AMS is visible onscreen:
++
+image::rpa-setup-image-search-wizard.png[RPA Builder Image Search Wizard]
+
+. Re-identify the field to enter the password. It may be necessary to add a similar Action Step previously for entering the user name in the xref:rpa-builder::toolbox-app-automation-keystrokes-to-app-element.adoc[Keystrokes to App Element Action Step].
```

Comment:
```suggestion
. Reidentify the field to enter the password. You might have to add a similar Action Step as previously for entering the user name in the xref:rpa-builder::toolbox-app-automation-keystrokes-to-app-element.adoc[Keystrokes to App Element Action Step].
```

Reply:
It is not clear what "previously" is referring to.

Reply:
Hi Diane,
the screenshot shows just one Action Step for entering the password for logging in to the AMS. If the username must be entered, too, the user/reader needs to put another _Keystrokes to App Element_ Action Step in front of/before the password-entering-step in the workflow.

Reply:
I'll add "Keystrokes to App Element" before "Action Step" to make this clearer.